examples/agent_motion_prediction/agent_motion_config.yaml

```diff
@@ -34,7 +34,7 @@ raster_params:
   # the keys are relative to the dataset environment variable
   satellite_map_key: "aerial_map/aerial_map.png"
   semantic_map_key: "semantic_map/semantic_map.pb"
-  dataset_meta_key: "semantic_map/meta.json"
+  dataset_meta_key: "meta.json"
```

Comment:
Are you going to add `ecef_to_ariel` to the meta.json here in aws (https://s3.console.aws.amazon.com/s3/buckets/lyft-av-prod-pdx-avresearch/magicpony/?region=us-west-2&tab=overview) and make a copy of it from in `sematic_map`'s parent folder?

Reply:
yes, I will upload the new meta.json :)